docs/development_client_site.rst

```diff
@@ -0,0 +1,32 @@
+Client-site Development
+=======================
+
+General approach
+----------------
+Falcon is using `Features module<https://drupal.org/project/features>`_ to
+package configurations.
+Please see `Features handbook<https://www.drupal.org/docs/8/modules/features>`_
+for detailed information on Features.
+
+When feature is installed for the first time its configuration files are
+getting imported into site configuration - once it is done then this
+configuration is owned by the site.
+
+It is recommended to use the `Config Distro<https://www.drupal.org/project/config_distro>`_
+module to manage configuration updates coming from Falcon distribution.
+
+How to install Falcon feature?
+------------------------------
+Go to `/admin/modules` and install it.
+
+How to update Falcon feature?
+-----------------------------
+After updating the codebase to the new version of Falcon go to
+`/admin/config/development/configuration/distro` and import feature updates.
```

Comment:
Is there a drush command that does it? I was thinking about writing some hook_update which would update configs automatically.

Reply:
There is the `config-distro-update` command. I would still start with the manual process and then improve it as we go.